src/main/java/edu/greenblitz/robotName/utils/tuneableNumber/TunableNumber.java

```diff
@@ -0,0 +1,49 @@
+package edu.greenblitz.robotName.utils.tuneableNumber;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import org.littletonrobotics.junction.Logger;
+
+public class TunableNumber {
```

Comment:
Have you thought about making this entire thing generic? I think think it would be that hard

Reply:
you can't. because the get you need it to get only primitive type, so you cant do this for object that contain more than 1 variable in themselves. and there is no reason to make it work on String"

Reply:
There is probably no need to use it for a string, but we might want to use it for an enum.
Either way, you can use a feature called Generics (https://theawesomenayak.com/java-101-generics) to make this class useable for each variable type.

Reply:
for enum is better to use list of option instead of  free writing to make sure there is no spelling problems

Reply:
Right.
If we can only use this for when you need to type the value, then there is no need for a more generic option.